src/app/services/xml-parser-service/xmp-parser.service.ts

```diff
@@ -0,0 +1,179 @@
+import {
+  org,
+  ManifestParser as manifestParser
+} from '@cruglobal/godtools-shared';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+export type Text = org.cru.godtools.shared.tool.parser.model.Text;
+export type Content = org.cru.godtools.shared.tool.parser.model.Content;
+export type Image = org.cru.godtools.shared.tool.parser.model.Image;
+export type Page = org.cru.godtools.shared.tool.parser.model.page.Page;
+export type CallToAction =
+  org.cru.godtools.shared.tool.parser.model.tract.CallToAction;
+export type Header = org.cru.godtools.shared.tool.parser.model.tract.Header;
+export type Hero = org.cru.godtools.shared.tool.parser.model.tract.Hero;
+export type Modal = org.cru.godtools.shared.tool.parser.model.tract.Modal;
+export type TractPage =
+  org.cru.godtools.shared.tool.parser.model.tract.TractPage;
+export type Card =
+  org.cru.godtools.shared.tool.parser.model.tract.TractPage.Card;
```

Comment:
so, I would probably export this as `TractPageCard` or put it under the `TractPage` namespace. This `Card` type only shows up as a top level element within a `TractPage`, and shouldn't be confused with the newer `Card` content element that can show up anywhere.

Reply:
some of this is just left over artifacts of how the spec has developed, There are 5 models that are only present on `TractPages` and not part of the common content elements. `Header`, `Hero`, `TractPage.Card`, `Modal`, and `CallToAction`.

Reply:
and then there are a few types are child types for single elements only. `Tabs.Tab`, `Accordion.Section`, `Flow.Item`, `Multiselect.Option`

Reply:
That's a good point. I've renamed it.